platforms/hyperledger-fabric/configuration/roles/create/cli_pod/tasks/main.yaml

```diff
@@ -29,7 +29,7 @@
   loop: "{{ peers }}"
   loop_control:
     loop_var: peer
-  when: peer.cli == "enabled"
+  when: peer.cli is defined and peer.cli == "enabled"
```

Comment:
add the same condition to git push task as well

Reply:
Added